src/utils/localStorage.ts

```diff
@@ -0,0 +1,2 @@
+export const accessTokenKey: (venueId: string) => string = (venueId) =>
+  `token-${venueId}`;
```

Comment:
I normally wouldn't bother typing so explicitly for something this simple. The inferred typing is good enough, and makes it more readable:

```suggestion
export const accessTokenKey = (venueId: string) => `token-${venueId}`;
```